VideoLocker/src/main/java/org/edx/mobile/util/MediaConsentUtils.java

```diff
@@ -14,6 +16,8 @@
  */
 public class MediaConsentUtils {
 
+    @Inject Config config;
```

Comment:
@aleffert: Since you have made this utility class non-static (why though?), if you have it injected everywhere, then RoboGuice will take care of injecting it's own fields. In a static utility class, injections can be performed statically as well from the `EdxDefaultModule`; I have done it for `BrowserUtil`. Note that the `Config` is explicitly passed as a parameter in `consentToMediaPlayback()` though; if you're having it injected, then this parameter can be removed.